index.html

```diff
@@ -6,5 +6,123 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <main>
+      <header>
+        <h1>Gesa Siebert</h1>
+        <nav>
+          <a href="" #biography>Biography</a>
+          <a href="#contact">Contact</a>
```

Comment:
Additionally you should use `ul` and `li` items here and wrap the `a` tag with one `li` iteam each.